cilium-health/launch/endpoint.go

```diff
@@ -184,8 +184,7 @@ func LaunchAsEndpoint(owner endpoint.Owner, hostAddressing *models.NodeAddressin
 	}
 
 	// Give the endpoint a security identity
-	_, errLabelsAdd := owner.UpdateSecLabels(info.ContainerID, lbl, labels.Labels{})
-	if errLabelsAdd != nil {
+	if errLabelsAdd := ep.ModifyIdentityLabels(owner, lbl, labels.Labels{}); err != nil {
```

Comment:
Is 'err' from the previous command intentionally checked here? or did you intend to check if `errLabelsAdd != nil` at the end of the line?

Reply:
fixed